.github/workflows/sync-from-fork.yml

```diff
@@ -18,9 +20,21 @@ jobs:
           git fetch upstream && git diff --name-only --diff-filter AM HEAD..remotes/upstream/master > updatedfiles.txt
       - name: Check for modified files
         id: git-check
-        run: echo ::set-output name=changes_present::$(if grep -qE "${FILES_TO_CHECK}" updatedfiles.txt; then echo 'true'; else echo 'false'; fi)
+        run: echo ::set-output name=changes-present::$(if grep -qE "${FILES_TO_CHECK}" updatedfiles.txt; then echo 'true'; else echo 'false'; fi)
       - name: show files
         run: cat updatedfiles.txt
       - name: Have files changed
-        if: steps.git-check.outputs.changes_present == 'false'
-        run: exit 1
\ No newline at end of file
+        if: steps.check-for-changes.outputs.changes-present == 'false'
+        run: echo "No changes needed to be synchronised"
+  sync:
```

Comment:
`sync-from-fork`

Reply:
The reference is to the job name, which in this case is 
```
jobs:
  check-for-changes:
```
The name in line one is just a friendly name for whole workflow, and the filename is distinct. No changes needed IMO.